workshops/login_auth/README.md

```diff
@@ -0,0 +1,501 @@
+---
+name: 'Web Login Authentication'
+description: 'Build a secure web login page using FIREBASE'
+author: '@tanishq-soni'
+---
+
+# Web Login Authentication 🔒
+
+In this workshop you'll build a secure web login page using [FIREBASE](https://firebase.google.com/).
+
+You will make something like this 👇
+
+![live gif](https://cloud-7782l8k04.vercel.app/loginv2.gif)
+
+[Live Example](https://loginauth-12.web.app)
+
+Try live example using:
+- EMAIL-`login@authentication.com`
+- PASSWORD-`password`
+
+## Getting Started :rocket:
+
+
+### Creating a FIREBASE project 💻 -
```

Comment:
`FIREBASE` should be `Firebase` every where

https://firebase.google.com/brand-guidelines